pkg/relayer/proxy/http_utils.go

```diff
@@ -78,14 +78,36 @@ func SafeReadBody(logger polylog.Logger, body io.ReadCloser, maxSize int64) ([]b
 
 	// Check if the body exceeded our size limit
 	if bytesRead > maxSize {
-		err := fmt.Errorf("❌ request body size exceeds maximum allowed body: %d bytes read > %d bytes limit", bytesRead, maxSize)
-		logger.Error().Err(err).Msg("❌ Request/response body too large")
-		return nil, err
+		return nil, ErrRelayerProxyMaxBodyExceeded.Wrapf(
+			"❌ body size exceeds maximum allowed body: %d bytes read > %d bytes limit",
```

Comment:
We should avoid having emojis in the error messages and prefer having them in the logs.